app/components/markdown/markdown_latex_block/index.tsx

```diff
@@ -147,45 +165,28 @@ const LatexCodeBlock = ({content, theme}: Props) => {
                 theme,
             });
         }
-    }, [managedConfig, intl, insets, theme]);
-
-    const splitContent = () => {
-        const lines = splitLatexCodeInLines(content);
-        const numberOfLines = lines.length;
-
-        if (numberOfLines > MAX_LINES) {
-            return {
-                content: lines.slice(0, MAX_LINES),
-                numberOfLines,
-            };
-        }
+    }, [managedConfig.copyAndPasteProtection, intl, insets, theme]);
```

Comment:
On the body of the callback you add the `?` to managedConfig, but not here. Either it is needed in both or in none.